### PR TITLE
Gamelog replay

### PR DESCRIPTION
**Pull request by asimmon**

### CODE REVIEW

**Comment by asimmon** [comment]
Closes #19

File: src/frontend/src/views/Replay.vue [hunk 1]
```diff
@@ -0,0 +1,214 @@
+<template>
+  <div class="replay">
+    <div v-bind:class="{ container: !isFullscreen, fullscreen: isFullscreen }">
+      <div class="row align-items-center">
+        <div class="col-auto mr-auto">
+          <h2 class="my-4">Replay a gamelog file</h2>
```
Comment (joseph-passineau) [inline src/frontend/src/views/Replay.vue]: I would have the name of the game. Maybe the name of the file or have some metadata of the game it self in the json.
Example: Replay game XHT4QDhjL (2019-08-14 9:14:32)

File: src/server/bombermanRoom.ts [hunk 2]
```diff
@@ -103,6 +103,8 @@ export class BombermanRoom extends TickBasedRoom<GameState> {
   }
 
   public applyGameAction(str: string) {
+    console.log(str);
```
Comment (joseph-passineau) [inline src/server/bombermanRoom.ts]: Probably want to remove this.

File: src/frontend/src/views/Replay.vue (hunk 1 above)
Comment (asimmon) [reply src/frontend/src/views/Replay.vue]: I've used "Replay {{filename}} <small>{{roomId}}</small"

File: .travis.yml [hunk 3]
```diff
@@ -8,5 +8,6 @@ script:
   - npm run test
   - npm run build
   - npm pack --dry-run
+  - npm run start:client:simulate
```
[the remark this answers is not in the document]
Comment (asimmon) [reply .travis.yml]: Done! The scripts are: 
```
clean, test
build, build-server, build-client, build-simulator
build-frontend-development, build-frontend-production
training-export, training-browser, join-match
frontend, server
```

**Pull request merged**